README.md

```diff
@@ -39,4 +36,36 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like developement mode (which also enables features like hot reloading when you make a file change).
 * There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
-When running `setup.sh`, the `.env.template` file will be copied to `.env` if the latter does not exist.
\ No newline at end of file
+When running `setup.sh`, the `.env.template` file will be copied to `.env` if the latter does not exist.
+
+### Run on Vagrant as VM
+```bash
+$ vagrant up
+$ vagrant provision
+$ vagrant up --provision
```

Comment:
You've included redundant commands with no explanation. `vagrant up` is all you need unless you want to re-run the provisioning script because of some changes.

Reply:
fixed